[2025-06-23T19:14:24.869+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: relatorio_revendas_diario.creating_csv manual__2025-06-23T19:14:08.393478+00:00 [queued]>
[2025-06-23T19:14:24.887+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: relatorio_revendas_diario.creating_csv manual__2025-06-23T19:14:08.393478+00:00 [queued]>
[2025-06-23T19:14:24.888+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-06-23T19:14:24.990+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): creating_csv> on 2025-06-23 19:14:08.393478+00:00
[2025-06-23T19:14:25.001+0000] {standard_task_runner.py:60} INFO - Started process 158242 to run task
[2025-06-23T19:14:25.009+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'relatorio_revendas_diario', 'creating_csv', 'manual__2025-06-23T19:14:08.393478+00:00', '--job-id', '118', '--raw', '--subdir', 'DAGS_FOLDER/DagEnviarRelatorioDesempenho.py', '--cfg-path', '/tmp/tmp0ew44ryr']
[2025-06-23T19:14:25.018+0000] {standard_task_runner.py:88} INFO - Job 118: Subtask creating_csv
[2025-06-23T19:14:25.167+0000] {task_command.py:423} INFO - Running <TaskInstance: relatorio_revendas_diario.creating_csv manual__2025-06-23T19:14:08.393478+00:00 [running]> on host 99f5714845f7
[2025-06-23T19:14:25.460+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='relatorio_revendas_diario' AIRFLOW_CTX_TASK_ID='creating_csv' AIRFLOW_CTX_EXECUTION_DATE='2025-06-23T19:14:08.393478+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-23T19:14:08.393478+00:00'
[2025-06-23T19:14:25.482+0000] {logging_mixin.py:188} INFO - Erro ao 
[2025-06-23T19:14:25.482+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-23T19:14:25.499+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=relatorio_revendas_diario, task_id=creating_csv, execution_date=20250623T191408, start_date=20250623T191424, end_date=20250623T191425
[2025-06-23T19:14:25.620+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-23T19:14:25.668+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
